solutions/devsprint-riccardo-1/OnboardingChallenge/Screens/EmptyView/EmptyView.swift

```diff
@@ -9,22 +9,29 @@ struct EmptyViewConfiguration {
 }
 
 class EmptyView: UIView {
-  private var titleLabel: UILabel = {
+  private let stackView: UIStackView = {
+    var stackView = UIStackView()
+    stackView.axis = .vertical
+    stackView.alignment = .center
+    stackView.spacing = 16
+    stackView.translatesAutoresizingMaskIntoConstraints = false
+    return stackView
+  }()
+
+  private let titleLabel: UILabel = {
     var title = UILabel()
     title.font = UIFont.boldSystemFont(ofSize: 18)
-    title.text = "No repositories found"
     title.textColor = .black
     title.textAlignment = .center
     title.numberOfLines = 0
     title.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
when working with `UIStackView`s, we don't have to manually set this boolean. The stack view will do it for us!